standalone/node/tests/common.rs

```diff
@@ -167,10 +167,10 @@ pub fn find_ws_url_from_output(read: impl Read + Send) -> (String, String) {
 			// does the line contain our port (we expect this specific output from substrate).
 			let sock_addr = match line.split_once("Running JSON-RPC WS server: addr=") {
 				None => return None,
-				Some((_, after)) => after.split_once(',').unwrap().0,
+				Some((_, after)) => after.split_once(",").unwrap().0,
 			};
 
-			Some(format!("ws://{sock_addr}"))
+			Some(format!("ws://{}", sock_addr))
```

Comment:
Strange changes in this file. Aren't the previous version better?

Reply:
I think I can revert this (but it merged...) khala-node test broken in my macOS again, so I just copy test from Substrate repo, didn't check what's changed

Reply:
> I think I can revert this (but it merged...) phala-node test broken in my macOS again, so I just copy test from Substrate repo, didn't check what's changed

Ah, I didn't aware that it was caused by our refactor before. Then it's good to keep it as is. No need to revert.